kinematics_interface_kdl/src/kinematics_interface_kdl.cpp

```diff
@@ -255,7 +256,7 @@ bool KinematicsInterfaceKDL::calculate_frame_difference(
   delta_x_ = KDL::diff(frames_(0), frames_(1), dt);
   for (size_t i = 0; i < 6; ++i)
```

Comment:
```suggestion
   for (auto i = 0; i < 6; ++i)
```
Can we use it this way?

Reply:
I don't think this will work? We need `Eigen::Index` and `int` in line 259?